src/Sulu/Bundle/ContentBundle/Resources/config/preview.xml

```diff
@@ -45,13 +45,29 @@
             <argument type="service" id="sulu_content.preview"/>
             <argument type="service" id="sulu_core.webspace.request_analyzer"/>
             <argument type="service" id="doctrine"/>
+            <argument type="service" id="sulu.content.mapper"/>
             <argument type="service" id="logger"/>
+
+            <tag name="sulu.websocket.message.handler" dispatcher="admin" alias="sulu_content.preview" />
         </service>
 
         <service id="sulu_content.preview.exception_listener" class="%sulu_content.preview.exception_listener.class%">
             <argument type="service" id="templating" />
+            <argument type="service" id="logger" />
             <argument type="string">%sulu.content.preview.error_template%</argument>
+
             <tag name="kernel.event_listener" event="kernel.exception" method="onKernelException" />
         </service>
+
+        <service id="sulu_content.jsconfig.preview" class="%sulu_content.jsconfig.preview.class%">
```

Comment:
in the new structure of `config` in javascript is that we have something like a parameter system and i have seperated the two concerns in two parameters i wouldnt put that two kind of things in symfony2 parameters!